drivers/sensor/ltc4162/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_LTC4162 ltc4162.c)
```

Comment:
No need to use ifdef here; see https://github.com/zephyrproject-rtos/zephyr/pull/36632.